src/olympia/devhub/tests/test_feeds.py

```diff
@@ -73,7 +73,7 @@ def test_dashboard(self):
         r = self.client.get(reverse('devhub.addons'))
         doc = pq(r.content)
         assert len(doc('li.item')) == 4
-        assert doc('.subscribe-feed').attr('href')[:-32] == (
+        assert doc('.subscribe-feed').attr('href')[:-36] == (
```

Comment:
I still have this explicit change on my list to check and re-check - it should be backwards compatible so this change should not happen.